src/main/resources/init_db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE IF NOT EXISTS `book_shop`;
+    CREATE TABLE `books` (
+        `id` bigint NOT NULL AUTO_INCREMENT,
+        `title` varchar(225),
+        `price` decimal,
```

Comment:
It's a good practice to define the precision and scale for the `decimal` type. For example, use `decimal(10, 2)` to store prices with up to 10 digits, including 2 decimal places.